composer.json

```diff
@@ -1,25 +1,66 @@
 {
-    "name": "robloach/libretro-netplay-registry",
-    "description": "libretro's NetPlay Registry",
+    "name": "ashura/libretro-netplay-registry",
```

Comment:
I created a symfony project from scratch and checked it in to the git. Seems like it took my username.